spec/controllers/stories/bulk_create_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Stories::BulkCreateController, type: :controller do
+  let(:project) { create :project }
+  let(:story_1) { attributes_for :story, project: project }
+  let(:story_2) { attributes_for :story, project: project }
+  let(:post_params) do
+    {
+      project_id: project.slug,
+      stories: [story_1, story_2]
+    }
+  end
+
+  context 'create stories to the project' do
+    before do
+      headers = { 'x-api-key': ENV['EXPORT_API_TOKEN'] }
+      request.headers.merge headers
+    end
+
+    subject { post :create, params: post_params }
+
+    it 'should create two new stories' do
```

Comment:
We can't (and shouldn't, anyway) assert that the stories were created because we don't make any assertion about it in the test, let's change the description:

```suggestion
    it 'should return created status' do
```

And also add tests that check that the operation was called (just check it was called, no need to check if `Story.count` changed) and also test the content of the response.